spec/mailers/acceptance_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "spec_helper"
+
+describe Acceptance do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I actually like leaving auto generated files because when running rspec locally you get reminded of the fact that certain aspects should be tested